lineman/app/components/mini_proposal_pie_chart/tiny_proposal_pie_chart.coffee

```diff
@@ -1,7 +1,8 @@
-angular.module('loomioApp').directive 'tinyProposalPieChart', ->
+angular.module('loomioApp').directive 'tinyProposalPieChart', ($timeout) ->
   scope: {proposal: '='}
   restrict: 'E'
   templateUrl: 'generated/components/mini_proposal_pie_chart/tiny_proposal_pie_chart.html'
   replace: true
   controller: 'MiniProposalPieChartController'
   link: (scope, element, attrs) ->
+    $timeout (-> scope.canLoadPie = true), 0
```

Comment:
This is a little hacky, but it works and I can't notice any performance penalty even on my slow-as machine.